resources/assets/js/forms.js

```diff
@@ -85,41 +97,49 @@ global.setHours = (hours) => {
   setInput(from, to);
 };
 
-$(function () {
+ready(function () {
   /**
      * Disable every submit button after clicking (to prevent double-clicking)
      */
-  $('form').submit(function (ev) {
-    $('input[type="submit"]').prop('readonly', true).addClass('disabled');
-    return true;
+  document.querySelectorAll('form').forEach((formElement) => {
+    formElement.addEventListener('submit', () => {
+      document.querySelectorAll('input[type="submit"],button[type="submit"]').forEach((element) => {
+        element.readOnly = true;
+        element.classList.add('disabled');
+      });
+      return true;
+    });
   });
-
 });
 
 /*
  * Button to set current time in time input fields.
  */
-$(function () {
-  $('.input-group.time').each(function () {
-    const elem = $(this);
-    elem.find('button').on('click', function () {
+ready(function () {
+  document.querySelectorAll('.input-group.time').forEach((element) => {
+    const button = element.querySelector('button');
+    if (!button) return;
+
+    button.addEventListener('click', () => {
       const now = new Date();
-      const input = elem.children('input').first();
-      input.val(formatTime(now));
-      const daySelector = $('#' + input.attr('id').replace('time', 'day'));
-      const days = daySelector.children('option');
+      const input = element.querySelector('input');
+      if (!input) return;
+
+      input.value = formatTime(now);
+      const daySelector = document.getElementById(input.id.replace('time', 'day'));
+      const dayElements = daySelector.querySelectorAll('option');
       const yyyyMMDD = formatDay(now);
-      days.each(function (i) {
-        if ($(days[i]).val() === yyyyMMDD) {
-          daySelector.val($(days[i]).val());
+      dayElements.forEach((dayElement) => {
+        if (dayElement.value === yyyyMMDD) {
+          daySelector.value = dayElement.value;
           return false;
         }
       });
     });
   });
 });
 
-$(function () {
+ready(function () {
```

Comment:
arrow-function?

Reply:
Mention this one time per PR is sufficient :wink: